Quake/r_world.c

```diff
@@ -397,6 +397,11 @@ static void R_FlushBatch (qboolean fullbright_enabled, qboolean alpha_test, qboo
 				constant_factor = -0.125f;
 				slope_factor = -0.125f;
 			}
+			else if (vulkan_globals.depth_format == VK_FORMAT_D16_UNORM_S8_UINT || vulkan_globals.depth_format == VK_FORMAT_D16_UNORM)
```

Comment:
Please revert, we don't use 16 bit depth at all anymore. Sorry, that was my bad.

Reply:
No worries, it was just a quick change :) Should I also remove the checks for the depth-only formats?